hive3/src/main/java/org/apache/iceberg/mr/hive/serde/objectinspector/IcebergTimestampObjectInspectorHive3.java

```diff
@@ -50,6 +50,7 @@ public LocalDateTime convert(Object o) {
   }
 
   @Override
+  @SuppressWarnings("JavaLocalDateTimeGetNano")
```

Comment:
Why is this ok to surpress, it does look like we are doing this incorrectly here, shouldn't this be Timestamp.valueOf(LocalDateTime time)?

I would separate this out into another PR since we could be doing a semantic change here

Reply:
fixed with Timestamp.valueOf(LocalDateTime time)